stable/mongodb-replicaset/values.yaml

```diff
@@ -29,6 +29,12 @@ image:
   repository: mongo
   tag: 3.6
   pullPolicy: IfNotPresent
+  ## Optionally specify an array of imagePullSecrets.
+  ## Secrets must be manually created in the namespace.
+  ## ref: https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/
+  ##
+  # pullSecrets:
```

Comment:
This is not specific to this image but applies to all images. Move to top level as `imagePullSecrets`.